web/handlers/invites.go

```diff
@@ -60,13 +54,88 @@ func (h inviteHandler) presentFacade(rw http.ResponseWriter, req *http.Request)
 
 	joinRoomURI.RawQuery = queryVals.Encode()
 
+	return template.URL(joinRoomURI.String())
+}
+
+func (h inviteHandler) presentFacade(rw http.ResponseWriter, req *http.Request) (interface{}, error) {
+	token := req.URL.Query().Get("token")
+
+	_, err := h.invites.GetByToken(req.Context(), token)
+	if err != nil {
+		if errors.Is(err, roomdb.ErrNotFound) {
+			return nil, weberrors.ErrNotFound{What: "invite"}
+		}
+		return nil, err
+	}
+
+	notice, err := h.pinnedNotices.Get(req.Context(), roomdb.NoticeDescription, "en-GB")
+	if err != nil {
+		return nil, fmt.Errorf("failed to find room's description: %w", err)
+	}
+
+	joinRoomURI := buildJoinRoomURI(h, token)
+
+	urlTo := web.NewURLTo(router.CompleteApp())
+	fallbackURL := urlTo(router.CompleteInviteFacadeFallback, "token", token)
+
+	// generate a QR code with the token inside so that you can open it easily in a supporting mobile app
+	qrCode, err := qrcode.New(joinRoomURI, qrcode.Medium)
```

Comment:
```suggestion
	qrCode, err := qrcode.New(string(joinRoomURI), qrcode.Medium)
````

this should do the trick.